From c1d504912fbbe920023441b77ef2f642db387e9a Mon Sep 17 00:00:00 2001
From: Kuninori Morimoto <kuninori.morimoto.gx@renesas.com>
Date: Thu, 23 Jan 2014 18:41:51 -0800
Subject: ASoC: rsnd: rsnd_ssi_probe() goes forwarder than rsnd_scu_probe()

rsnd_ssi_probe() goes forwarder than rsnd_scu_probe(),
since scu will need ssi information on Gen2

Signed-off-by: Kuninori Morimoto <kuninori.morimoto.gx@renesas.com>
Signed-off-by: Mark Brown <broonie@linaro.org>
(cherry picked from commit eb854f6dff24a59378acc8d8eda57a3543a25acc)
Signed-off-by: Simon Horman <horms+renesas@verge.net.au>
---
 sound/soc/sh/rcar/core.c | 6 +++---
 1 file changed, 3 insertions(+), 3 deletions(-)

diff --git a/sound/soc/sh/rcar/core.c b/sound/soc/sh/rcar/core.c
index b5af6f5145ea..f316a663e4d3 100644
--- a/sound/soc/sh/rcar/core.c
+++ b/sound/soc/sh/rcar/core.c
@@ -763,15 +763,15 @@ static int rsnd_probe(struct platform_device *pdev)
 	if (ret)
 		return ret;
 
-	ret = rsnd_scu_probe(pdev, info, priv);
+	ret = rsnd_ssi_probe(pdev, info, priv);
 	if (ret)
 		return ret;
 
-	ret = rsnd_adg_probe(pdev, info, priv);
+	ret = rsnd_scu_probe(pdev, info, priv);
 	if (ret)
 		return ret;
 
-	ret = rsnd_ssi_probe(pdev, info, priv);
+	ret = rsnd_adg_probe(pdev, info, priv);
 	if (ret)
 		return ret;
 
-- 
2.1.2

